docs/source/tutorial.rst

```diff
@@ -94,51 +96,58 @@ Now let’s get started! First, the general imports:
 
 
 
-One-dimensional integration
-----------------------------
+One-dimensional integration with complex numbers
```

Comment:
```suggestion
One-dimensional integration
```
the example needn't be complex so I would not make it seem as if the example is exclusively complex